File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentRepository.java
package com.metaengine.demo.shipping.services;

import com.metaengine.demo.shipping.aggregates.Shipment;
import java.util.List;

/** Persistence gateway for the Shipment aggregate. */
public class ShipmentRepository {

    /** Persist a new Shipment from a partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Shipment by id; returns null if not found. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Shipments, capped at the supplied limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/shipping/services/shipment_repository.py
"""ShipmentRepository for the shipping domain."""
from typing import Optional

from metaengine_demo.shipping.aggregates.shipment import Shipment


class ShipmentRepository:
    """ShipmentRepository persistence service."""

    def create(self, input: Optional[Shipment]) -> Shipment:
        """Persist a new shipment."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Shipment]:
        """Find a shipment by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Shipment]:
        """List shipments up to a limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a shipment by id."""
        raise NotImplementedError("not implemented")
